Sources/NeoBrutalism/Components/Progress.swift

```diff
@@ -9,28 +9,33 @@ public struct NBProgressViewStyle: ProgressViewStyle {
 
     public func makeBody(configuration: Configuration) -> some View {
         let value = configuration.fractionCompleted ?? 0.0
-        GeometryReader { geometry in
-            HStack(spacing: 0) {
-                // Progress bar
-                Rectangle()
-                    .fill(theme.main)
-                    .frame(width: value * geometry.size.width, height: theme.size)
-
-                if value > 0.001 && value < 0.99 {
-                    Divider()
-                        .frame(width: theme.borderWidth, height: geometry.size.height)
-                        .background(Color.black)
+        HStack {
```

Comment:
We should use the spacing from theme instead of default one so users can easily customize them.

Reply:
Spacing for the HStack, padding, or frame?

Reply:
removed default spacing for HStack

Reply:
Thanks! (I might make more changes to this later as needed.)